x/programs/rust/examples/token/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-wasmlanche_sdk = { version = "0.1.0", path = "../../wasmlanche_sdk" }
+wasmlanche_sdk = { version = "0.1.0", path = "../../wasmlanche_sdk", features = ["simulator"] }
```

Comment:
🤔  @hexfusion, instead of having a `simulator` feature, you could probably just throw the mod behind `cfg(test)` so that it's always available for testing. Thoughts?

@exdx (not a comment for changes on this PR)

Reply:
> 🤔 @hexfusion, instead of having a simulator feature, you could probably just throw the mod behind cfg(test) so that it's always available for testing. Thoughts?

As long as we doc that I think it would be fine. Can always add back feature if folks want that.